lms/djangoapps/verify_student/views.py

```diff
@@ -195,6 +197,19 @@ class PayAndVerifyView(View):
     VERIFICATION_DEADLINE = "verification"
     UPGRADE_DEADLINE = "upgrade"
 
+    def _get_user_active_status(self, user):
```

Comment:
should this be named to be specific about the fact that this should only be called in the context of a learner paying for verified? or is the fact that it is within this view enough?